luigi_pipeline/lib/hail_tasks.py

```diff
@@ -48,6 +48,10 @@ class HailMatrixTableTask(luigi.Task):
     genome_version = luigi.Parameter(description='Reference Genome Version (37 or 38)')
     vep_runner = luigi.ChoiceParameter(choices=['VEP', 'DUMMY'], default='VEP', description='Choice of which vep runner'
                                                                                             'to annotate vep.')
+    ignore_missing_samples = luigi.BoolParameter(default=False, description="Allow missing samples in the callset.")
+    skip_sample_subset = luigi.BoolParameter(default=False, description=
+        "Skip subsetting samples... only subset on present variants."
```

Comment:
I don't think there's any reason no!  I think even python linters are pretty lenient on single/double quotes since both are supported.  I will change these to single quotes.

I saw some double quotes floating around: https://github.com/broadinstitute/seqr-loading-pipelines/blob/master/luigi_pipeline/seqr_loading.py#L70-L73, so didn't try hard to be consistent but do think it's important!